src/store/store-config.js

```diff
@@ -0,0 +1,16 @@
+import {
+  createStore,
+  combineReducers,
+  applyMiddleware,
+  compose,
+} from 'redux';
+import thunk from 'redux-thunk';
+
+const composeEnhancers = window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__ || compose; // eslint-disable-line no-undef
```

Comment:
It’s old approach to add dev tools, read the doc please and forget about vtb project :)